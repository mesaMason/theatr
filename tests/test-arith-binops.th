func main() -> int:
    double f;
    print(1 + 1);
    f = 0.5 + 0.5;
    print(f);
    f = .0 + 0.1;
    print(f);
    f = -.9 + .1;
    print(f);
    print(2 - 5);
    print(3 * 5);
    print(4 / 5);
    return 0;
