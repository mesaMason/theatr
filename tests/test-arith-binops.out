2
1.000000
0.100000
-0.800000
-3
15
0
